src/content/facts/09-cannibalism.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Sex sometimes turns into cannibalism.
+source: BBC Earth
+sourceUrl: http://www.bbc.com/earth/story/20150223-mysteries-of-cannibal-octopus-sex
+image: /images/valvos.jpg
+imageSource: https://www.oldbookillustrations.com/illustrations/valvos/
+---
+
+Octopus sex, as you might anticipate from a creature with too many arms, is a bit of an odd proposition. For male octopuses, it's also quite a dangerous one.
```

Comment:
> too many arms

Maybe "so many" or "that many" instead? I don't want to knock them just because they have so many arms!

```suggestion
Octopus sex, as you might anticipate from a creature with that many arms, is a bit of an odd proposition. For male octopuses, it's also quite a dangerous one.
```

Reply:
I've done some copyediting; let me know if this reads better now:

<img width="512" alt="Screenshot 2019-11-08 at 16 34 13" src="https://user-images.githubusercontent.com/376315/68494116-bface700-0245-11ea-9575-2f0eb0b1f838.png">